package/src/bin/omicron-package.rs

```diff
@@ -242,19 +176,18 @@ async fn get_sha256_digest(path: &PathBuf) -> Result<Digest> {
 }
 
 // Accesses a package which is contructed outside of Omicron.
```

Comment:
This function now handles local/composite packages too.

Reply:
Updated in 3b469a56d646e2d827d6b880c0db84e0b65537c1